jsonschema-core.xml

```diff
@@ -1830,6 +1820,74 @@
                 </section>
             </section>
 
+            <section title="Compound Documents">
+                <t>
+                    A Compound Schema Document is defined as JSON document (sometimes called a "bundled" schema)
+                    which has multiple embedded JSON Schema Resources bundled into the same document to
+                    ease transportation.
+                </t>
+                <t>
+                    Each embedded Schema Resource MUST be treated as an individual Schema Resource, following standard
+                    schema loading and processing requirements, including determining voacbulary support.
+                </t>
+                <section title="Bundling">
+                    <t>
+                        A Compound Schema Document is created by taking references (such as "$ref")
+                        to an external Schema Resource and embedding the referenced Schema Resources
+                        within the referreing document.
+                    </t>
+                    <t>
+                        Each embedded JSON Schema Resource (and the document's root) MUST identify itself
+                        with an absolute URI using the "$id" keyword, and SHOULD make use of the "$schema"
+                        keyword to identify the dialect it is using, in the root of the schema resource.
+                    </t>
+                    <t>
+                        When a by-reference applicator is dereferenced as part of the bundling process, the Schema Resource
+                        MUST NOT be embedded by replacing the schema object from which it was referenced, or by wrapping
+                        in other applicator keywords. Instead, a bundled Schema Resource MUST be located as a value of
+                        a "$defs" object at the containing schemas root. The key of the "$defs" for the now embedded
+                        Schema Resource MAY be the "$id" of the bundled schema or some other form of application defined
+                        unique identifer (such as a UUID). This key is not intended to be referenced in JSON Schema, but may
+                        be used by an application to aid the bundling process.
+                    </t>
+                    <t>
+                        References in the containing schema document to the previously external Schema Resources
+                        MUST NOT be changed, and now resolve to a schema using the "$id" of an embedded Schema Resource.
+                    </t>
+                </section>
+                <section title="Differing and Default Dialects">
+                    <t>
+                        In addition to the rules defining the use of
+                        <xref target="keyword-schema">the "$schema" keyword</xref>,
```

Comment:
Is that overly verbose? I felt that since additional handling of $schema is being defined, it wouldn't hurt to remind people that some rules exist already.

Reply:
I just don't understand how it connects to the rest of the sentence.  To me it reads like to kinda-tangentially-related things stuck together.  The rest of the sentence is clear on its own, but this one implies that I'm probably forgetting something super-relevant in some other section and I have no idea what it is.  Why do other parts of the `$schema` rules have particular relevance here?

Reply:
Actually, you're right. There's no need for this. I've removed the phrasing. Pushed.